docs/usage/PRODUCT_SYNC.md

```diff
@@ -59,14 +59,12 @@ order for the sync to resolve the actual ids of those references, those `key`s h
 final ProductSyncOptions productSyncOptions = ProductSyncOptionsBuilder.of(sphereClient).build();
 ````
 
-The options can be used to provide additional optional configuration for the sync as well:
+Additional optional configuration for the sync can be configured on the `ProductSyncOptionsBuilder` instance, according to your need:
```

Comment:
I think you don`t need "according to your need" here

Reply:
I just wanted to send through the idea that everyone has different needs that can be achieved via different settings in the options.